tests/checkpointing/test_model_checkpoint.py

```diff
@@ -109,7 +109,6 @@ def training_step(self, batch, batch_idx):
         def validation_step(self, batch, batch_idx):
             log_value = self.val_logs[self.current_epoch, batch_idx]
             self.log("val_log", log_value)
-            self.log("epoch", self.current_epoch, on_epoch=True)
```

Comment:
Since we use `setdefault` now inside `ModelCheckpoint`, this `log` call would set the `epoch` value but it would be a float so the filename would be `epoch=1.0.ckpt`.

Anyways, we don't need to log it anymore

Reply:
> but it would be a float so the filename would be

How do you feel about filenames having epoch numbers as floating points? Should we just cast them to integers instead?

Reply:
Probably it's best to do it so accidents are avoided.